src/components/swap/AdvancedSwapDetails.tsx

```diff
@@ -9,7 +9,7 @@ import { ZERO_PERCENT } from 'constants/misc'
 import useNativeCurrency from 'lib/hooks/useNativeCurrency'
```

Comment:
A few more spots in the AdvancedSwapDetails are missing currency conversion
![Screenshot 2023-09-06 at 9 42 13 AM](https://github.com/Uniswap/interface/assets/11512321/373e2737-386d-43f8-8349-2cb4dc9ee6dc)
![Screenshot 2023-09-06 at 9 42 32 AM](https://github.com/Uniswap/interface/assets/11512321/17f9b3cd-025c-4977-98a1-800851a07f4a)

Reply:
these don't use the formatCurrencyAmount function